lib/common/bitstream.h

```diff
@@ -220,11 +222,14 @@ MEM_STATIC size_t BIT_initCStream(BIT_CStream_t* bitC,
 }
 
 /*! BIT_addBits() :
- *  can add up to 26 bits into `bitC`.
+ *  can add up to 31 bits into `bitC`.
  *  Note : does not check for register overflow ! */
 MEM_STATIC void BIT_addBits(BIT_CStream_t* bitC,
                             size_t value, unsigned nbBits)
 {
+    MEM_STATIC_ASSERT(sizeof(BIT_mask) / sizeof(BIT_mask[0]) == 32);
```

Comment:
Since the constant `sizeof(BIT_mask) / sizeof(BIT_mask[0]` comes back often, you could hide it in a `#define` ?
It would also help to understand what it means.